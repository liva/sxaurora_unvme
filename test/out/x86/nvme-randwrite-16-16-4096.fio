[global]
norandommap=1
thread=1
group_reporting=1
direct=1
verify=0
ioengine=psync
time_based=1
ramp_time=30
runtime=60
bs=4096
filename=/dev/nvme0n1

[test]
rw=randwrite
numjobs=16
iodepth=16
