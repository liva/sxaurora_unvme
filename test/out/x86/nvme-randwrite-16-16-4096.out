# /bin/uname -a (Thu 06/06/19 04:45:50 PM)
Linux aurora6 3.10.0-957.5.1.el7.x86_64 #1 SMP Fri Feb 1 14:54:57 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

# cat /home/sawamoto/vepci/test/out/nvme-randwrite-16-16-4096.fio (Thu 06/06/19 04:45:50 PM)
[global]
norandommap=1
thread=1
group_reporting=1
direct=1
verify=0
ioengine=psync
time_based=1
ramp_time=30
runtime=60
bs=4096
filename=/dev/nvme0n1

[test]
rw=randwrite
numjobs=16
iodepth=16


========

# /usr/bin/free -h (Thu 06/06/19 04:45:50 PM)
              total        used        free      shared  buff/cache   available
Mem:            93G        2.7G         79G         34M         10G         89G
Swap:          4.0G          0B        4.0G


========

# /home/sawamoto/fio_x86/fio /home/sawamoto/vepci/test/out/nvme-randwrite-16-16-4096.fio (Thu 06/06/19 04:45:50 PM)
test: (g=0): rw=randwrite, bs=(R) 4096B-4096B, (W) 4096B-4096B, (T) 4096B-4096B, ioengine=psync, iodepth=16
...
fio-2.19
Starting 16 threads

test: (groupid=0, jobs=16): err= 0: pid=120314: Thu Jun  6 16:48:52 2019
  write: IOPS=361k, BW=1412MiB/s (1480MB/s)(82.8GiB/60002msec)
    clat (usec): min=20, max=18471, avg=42.82, stdev=63.11
     lat (usec): min=21, max=18471, avg=43.04, stdev=63.11
    clat percentiles (usec):
     |  1.00th=[   27],  5.00th=[   28], 10.00th=[   29], 20.00th=[   30],
     | 30.00th=[   32], 40.00th=[   35], 50.00th=[   37], 60.00th=[   38],
     | 70.00th=[   39], 80.00th=[   41], 90.00th=[   49], 95.00th=[   77],
     | 99.00th=[  201], 99.50th=[  286], 99.90th=[  532], 99.95th=[  652],
     | 99.99th=[  940]
    lat (usec) : 50=90.35%, 100=6.21%, 250=2.79%, 500=0.53%, 750=0.09%
    lat (usec) : 1000=0.02%
    lat (msec) : 2=0.01%, 10=0.01%, 20=0.01%
  cpu          : usr=7.32%, sys=44.01%, ctx=21725514, majf=0, minf=1
  IO depths    : 1=100.0%, 2=0.0%, 4=0.0%, 8=0.0%, 16=0.0%, 32=0.0%, >=64=0.0%
     submit    : 0=0.0%, 4=100.0%, 8=0.0%, 16=0.0%, 32=0.0%, 64=0.0%, >=64=0.0%
     complete  : 0=0.0%, 4=100.0%, 8=0.0%, 16=0.0%, 32=0.0%, 64=0.0%, >=64=0.0%
     issued rwt: total=0,21686918,0, short=0,0,0, dropped=0,0,0
     latency   : target=0, window=0, percentile=100.00%, depth=16

Run status group 0 (all jobs):
  WRITE: bw=1412MiB/s (1480MB/s), 1412MiB/s-1412MiB/s (1480MB/s-1480MB/s), io=82.8GiB (88.9GB), run=60002-60002msec

Disk stats (read/write):
  nvme0n1: ios=43/21617234, merge=0/0, ticks=8/504859, in_queue=621536, util=39.21%


========

# /usr/bin/free -h (Thu 06/06/19 04:48:52 PM)
              total        used        free      shared  buff/cache   available
Mem:            93G        2.7G         79G         34M         10G         89G
Swap:          4.0G          0B        4.0G

